app/routers/kis_domestic_trading.py

```
@@ -0,0 +1,191 @@
+                "analysis_confidence": analysis.confidence if analysis else None,
+            })
+
+        return {
+            "success": True,
+            "krw_balance": krw_balance,
+            "total_stocks": len(processed_stocks),
+            "stocks": processed_stocks
+        }
+
+    except Exception as e:
+        logger.error(f"Error in get_my_domestic_stocks: {str(e)}", exc_info=True)
```

Ruff (0.14.6)

Use explicit conversion flag

Replace with conversion flag

(RUF010)